notebooks/hybrid_rag_pipeline_with_breakpoints.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Resume from a saved state
</pre><p>can be removed I guess</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>